app/uk/gov/hmrc/pensionschemereturn/transformations/nonsipp/MemberPaymentsTransformer.scala

```diff
@@ -39,23 +35,11 @@ class MemberPaymentsTransformer @Inject()(
   override def toEtmp(memberPayments: MemberPayments): EtmpMemberPayments =
     EtmpMemberPayments(
       recordVersion = memberPayments.recordVersion,
-      employerContributionMade = (
-        memberPayments.employerContributionsDetails.made,
-        memberPayments.memberDetails.flatMap(_.employerContributions).size,
-        memberPayments.employerContributionsDetails.completed
-      ) match {
-        // Completed with 0 contributions: transform to false, as both cases are logically equivalent
-        case (true, 0, true) => Some(false)
-        // Completed with 1+ contributions: no transformation required
-        case (true, _, true) => Some(true)
-        // In Progress with 0 contributions: no transformation required
-        case (true, 0, false) => Some(true)
-        // In Progress with 1+ contributions: transform to false, to differentiate from Completed with 1+ contributions
-        case (true, _, false) => Some(false)
-        // Not Started: transform to None
-        case (false, 0, false) => None
-        // (Not a valid state, but included for completeness)
-        case (answer, _, _) => Some(answer)
+      employerContributionMade = memberPayments.employerContributionsDetails match {
+        case SectionDetails(made @ true, completed @ true) => Some(true)
+        case SectionDetails(made @ true, completed @ false) => Some(true)
```

Comment:
This is now different to how it was before, if wont result in `Some(false)` in case of In Progress with 1+ contributions. Is that intentional?